synapse/handlers/room_member.py

```diff
@@ -258,7 +272,9 @@ async def ratelimit_invite(
         if room_id:
             await self._invites_per_room_limiter.ratelimit(requester, room_id)
 
-        await self._invites_per_user_limiter.ratelimit(requester, invitee_user_id)
+        await self._invites_per_recipient_limiter.ratelimit(requester, invitee_user_id)
+        if requester is not None:
+            await self._invites_per_issuer_limiter.ratelimit(requester, requester.user)
```

Comment:
```suggestion
            await self._invites_per_issuer_limiter.ratelimit(requester)
```
No point having a key in the ratelimiter if it's always the same as the requester, I think.